miri-script/src/commands.rs

```diff
@@ -35,26 +37,27 @@ impl MiriEnv {
         self.build(path!(self.miri_dir / "Cargo.toml"), &[], quiet)?;
         self.build(&manifest_path, &[], quiet)?;
 
-        let target = &match self.sh.var("MIRI_TEST_TARGET") {
-            Ok(target) => vec!["--target".into(), target],
-            Err(_) => vec![],
-        };
+        let target_flag =
+            &if let Some(target) = target { vec!["--target", target] } else { vec![] }[..];
```

Comment:
Why is `[..]` needed now? It wasn't needed before

Reply:
Not needed, I just like slices better

Reply:
I find it to be just syntactic noise, please remove it.